polkadot/runtime/parachains/src/scheduler/tests.rs

```diff
@@ -724,14 +741,15 @@ fn schedule_schedules_including_just_freed() {
 fn schedule_clears_availability_cores() {
 	let mut config = default_config();
 	config.scheduling_lookahead = 1;
+	config.on_demand_cores = 2;
```

Comment:
Why do we care about this number here?

Reply:
Right, the defaults are fine for this test.